task/sast-coverity-check/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://go.k8s.io/owners
```

Comment:
We have switched from OWNERS to CODEOWNERS (in the repo root), could you update this accordingly?

Reply:
Good point, removed.